<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AHP</class>
 <widget class="QMainWindow" name="AHP">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>750</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>AHP</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="0" colspan="2">
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="usesScrollButtons">
       <bool>true</bool>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <property name="tabBarAutoHide">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="input">
       <attribute name="title">
        <string>Input</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QGridLayout" name="inputLayout">
          <property name="leftMargin">
           <number>6</number>
          </property>
          <property name="topMargin">
           <number>6</number>
          </property>
          <property name="rightMargin">
           <number>6</number>
          </property>
          <property name="bottomMargin">
           <number>6</number>
          </property>
          <property name="horizontalSpacing">
           <number>0</number>
          </property>
          <item row="7" column="0" colspan="2">
           <widget class="QGroupBox" name="outputgroupBox">
            <property name="toolTip">
             <string/>
            </property>
            <property name="title">
             <string>Output</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QProgressBar" name="mainprogressBar">
               <property name="maximum">
                <number>1</number>
               </property>
               <property name="value">
                <number>0</number>
               </property>
               <property name="textVisible">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="outputLineEdit">
               <property name="toolTip">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;LSAT will create two files with the given name.&lt;/p&gt;&lt;p&gt;*name*.tif in \results\AHP\rasters\ and&lt;/p&gt;&lt;p&gt;*name*.npz in \results\AHP\tables\&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="methodpriorityComboBox">
               <property name="toolTip">
                <string>Method to derive the priority vector</string>
               </property>
               <item>
                <property name="text">
                 <string>Mean of the Row (MOTR)</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Row geometric mean method (RGMM)</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Eigenvalue method (EVM)</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="applyPushButton">
               <property name="text">
                <string>Apply</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QGroupBox" name="inputgroupBox">
            <property name="title">
             <string>Input Rasters</string>
            </property>
            <layout class="QGridLayout" name="gridLayout">
             <item row="2" column="1">
              <widget class="QToolButton" name="addrasterPushButton">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QToolButton" name="removerasterPushButton">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="1" column="0" rowspan="4">
              <widget class="QTableWidget" name="inputTableWidget">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="layoutDirection">
                <enum>Qt::LeftToRight</enum>
               </property>
               <property name="autoFillBackground">
                <bool>false</bool>
               </property>
               <property name="lineWidth">
                <number>1</number>
               </property>
               <property name="alternatingRowColors">
                <bool>true</bool>
               </property>
               <property name="textElideMode">
                <enum>Qt::ElideLeft</enum>
               </property>
               <property name="rowCount">
                <number>0</number>
               </property>
               <property name="columnCount">
                <number>3</number>
               </property>
               <attribute name="horizontalHeaderVisible">
                <bool>true</bool>
               </attribute>
               <attribute name="horizontalHeaderCascadingSectionResizes">
                <bool>false</bool>
               </attribute>
               <column>
                <property name="text">
                 <string>Parameter</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Unique values count</string>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Unique values</string>
                </property>
               </column>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="raster">
       <attribute name="title">
        <string>Pairwise raster comparison</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QGroupBox" name="rastergroupBox">
          <property name="title">
           <string>Pairwise Raster Comparison</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QTableWidget" name="rastertableWidget"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="backPushButton">
      <property name="text">
       <string>&lt; Back</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="nextPushButton">
      <property name="text">
       <string>Next &gt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>backPushButton</tabstop>
  <tabstop>nextPushButton</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>inputTableWidget</tabstop>
  <tabstop>addrasterPushButton</tabstop>
  <tabstop>removerasterPushButton</tabstop>
  <tabstop>outputLineEdit</tabstop>
  <tabstop>methodpriorityComboBox</tabstop>
  <tabstop>applyPushButton</tabstop>
  <tabstop>rastertableWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
